src/auth/auth.service.ts

```diff
@@ -134,11 +134,13 @@ export class AuthService {
             maxAge: AT_MAX_AGE * 1000,
             httpOnly: true,
             secure: true,
+            sameSite: "none", // TODO: remove this in future when we use actual domains for the dashboard, they will likely be same-site
```

Comment:
```suggestion
            sameSite: "none",
```

Reply:
remove this for now and put on click up instead, I need to do more research on how it is usually handled. 

I'm thinking set cookies based on the environment, i.e. NODE_ENV